gcc-4.8/build.sh

```diff
@@ -35,7 +35,9 @@ else
     # For reference during post-link.sh, record some
     # details about the OS this binary was produced with.
     mkdir -p "${PREFIX}/share"
-    cat /etc/*-release > "${PREFIX}/share/conda-gcc-build-machine-os-details"
+	# lsb_release can complain about LSB mosules in stderr, so we
+	# ignore that.
+    lsb_release -a 1> "${PREFIX}/share/conda-gcc-build-machine-os-details"
```

Comment:
This line looks reasonable, assuming `lsb_release` is always available on all Linux distros.  Is it?  BTW, if you're changing it here in `build.sh`, you should also change it in `post-link.sh`.